inc/class-wpseo-admin-bar-menu.php

```diff
@@ -470,61 +511,56 @@ protected function get_title() {
 	/**
 	 * Gets the current post if in a singular post context.
 	 *
-	 * @global string       $pagenow Current page identifier.
+	 * @return bool|int False when not singular, Post ID when it is.
+	 *
 	 * @global WP_Post|null $post    Current post object, or null if none available.
 	 *
-	 * @return WP_Post|null Post object, or null if not in singular context.
+	 * @global string       $pagenow Current page identifier.
```

Comment:
As far as I know, `@global` notations should come before the `@return` notation.

Reply:
Fixed.